src/app/base/components/VaultNotification/VaultNotification.test.tsx

```diff
@@ -20,6 +24,17 @@ it("displays a notification when data has loaded", async () => {
   const state = rootStateFactory();
   state.controller.loaded = true;
   state.general.vaultEnabled.loaded = true;
+  state.general.vaultEnabled.data = false;
```

Comment:
Might be worth adding a test that no notification is displayed when the vault is enabled.